### PR TITLE
Feature/585 Logging participants and accounts activation and deactivation

### PR DESCRIPTION
**Pull request by ggrg**
[Story #585](https://github.com/mojaloop/project/issues/585)

### CODE REVIEW

File: src/admin/participants/handler.js [hunk 1]
```diff
@@ -161,6 +162,11 @@ const update = async function (request) {
   Sidecar.logRequest(request)
   try {
     const updatedEntity = await Participant.update(request.params.name, request.payload)
+    if (request.payload.isActive !== undefined) {
+      const isActiveText = request.payload.isActive ? 'activated' : 'disabled'
```
Comment (rmothilal) [inline src/admin/participants/handler.js]: Should this be hardcoded or put into the enum?

File: src/admin/participants/handler.js [hunk 2]
```diff
@@ -312,6 +318,11 @@ const updateAccount = async function (request, h) {
       ledgerAccountType: await request.server.methods.enums('ledgerAccountType')
     }
     await Participant.updateAccount(request.payload, request.params, enums)
+    if (request.payload.isActive !== undefined) {
+      const isActiveText = request.payload.isActive ? 'activated' : 'disabled'
```
Comment (rmothilal) [inline src/admin/participants/handler.js]: Should this be hardcoded or pout into the enum?

File: src/admin/participants/handler.js (hunk 1 above)
Comment (ggrg) [reply src/admin/participants/handler.js]: I see your point and I've created local enum object. I don't think we should flood the global Enum with debugging messages, but probably have a story in future where we isolate all debug messages into a single file, which would later ease i18n. @elnyry, please backlog if appropriate.

File: src/admin/participants/handler.js (hunk 2 above)
Comment (ggrg) [reply src/admin/participants/handler.js]: same as previous

**Pull request merged**